src/NuGet.Clients/NuGet.PackageManagement.UI/PackageInstallationInfo.cs

```diff
@@ -156,7 +158,13 @@ public override int GetHashCode()
 
         public override string ToString()
         {
-            return $"{ProjectName} {InstalledVersion?.Version.ToString() ?? Resources.Text_NotInstalled}";
+            var vulnerabilityString = string.Empty;
+            if (InstalledVersionMaxVulnerability != -1)
+            {
+                vulnerabilityString = string.Format(CultureInfo.CurrentCulture, Resources.Label_PackageVulnerableToolTip, (PackageVulnerabilitySeverity)InstalledVersionMaxVulnerability);
```

Comment:
The InstalledVersionMaxVulnerability enum values are not localized. We use IntToVulnerabilitySeverityConverter in the XAML to convert to a localized string. We should consider localization for this accessibility fix.

Reply:
If im not wrong when we use CultureInfo it localizes the string.

I updated the code to use the converter, not sure if this is the best way to do it. We could also copy the code from the converter into this file.